app/src/main/res/layout/activity_login.xml

```diff
@@ -4,10 +4,10 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".user.LoginActivity">
+    tools:context=".LoginActivity">
```

Comment:
Some hardcoded strings also here.

Can use textEmail as inputType for the email address field.